### PR TITLE
SW-1739 Replace Hardcoded Colors

### PR DESCRIPTION
**Pull request by charlie-tf**
- Part 1 of ??

working spreadsheet: [here](https://docs.google.com/spreadsheets/d/1nAaoqbf8DoSoLI389HYXHcZya3NnzG1RFUeYSBr3GQA/edit#gid=324615937)

### CODE REVIEW

File: src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx [hunk 1]
```diff
@@ -51,10 +64,6 @@ export default function ViewViabilityTestModal(props: ViewViabilityTestModalProp
     onEdit();
   };
 
-  const titleStyle = {
-    color: '#859799',
```
Comment (karthikbtf) [inline src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx:55]: you can keep this as-is.

```
import { ..., useTheme } from '@mui/material';
...
const theme = useTheme();
...
const titleStyle = {
  color: theme.palette.TwClrTxtSecondary,
};
```
i believe `sx` is preferred over `makeStyles/className` where feasible - such as here in the previous version.
something to do with mui v5 etc. Constanza knows more about this preferred usage.

File: src/components/common/EmptyMessage.tsx [hunk 2]
```diff
@@ -73,10 +85,10 @@ export default function EmptyMessage(props: EmptyMessageProps): JSX.Element {
           {rowItems.map((rowItem, index) => (
             <div className={classes.rowItem} key={index}>
               <div className={classes.rowItemInfo}>
-                <Typography fontSize='16px' fontWeight={500} color='#000000' lineHeight='20px'>
+                <Typography className={classes.rowItemInfoTitle}>
                   {rowItem.title}
                 </Typography>
-                <Typography fontSize='14px' fontWeight={500} color='#3A4445' lineHeight='20px'>
```
Comment (karthikbtf) [inline src/components/common/EmptyMessage.tsx]: same here, leverage `useTheme` and you can access theme directly here

File: src/theme.ts [hunk 3]
```diff
@@ -3,5 +3,6 @@ import { theme } from '@terraware/web-components';
 // This import is needed to get the types for customized Palette, unfortunately this is the easiest fix for now.
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import TerrawareThemeOptions from '@terraware/web-components/style-dictionary-dist/TerrawareTheme';
+import TerrawareThemeOptions2 from '@terraware/web-components/style-dictionary-dist2/TerrawareTheme2';
```
Comment (karthikbtf) [inline src/theme.ts:6]: a million thanks 🙏 for keeping this in mind

**Review by karthikbtf — COMMENTED** [review]
@charlie-tf this is super awesome,, exciting to see the plans coming to life!
Thank you so much!
There are a few comments from me about keeping `sx` usage and leveraging `useTheme`.

File: src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx (hunk 1 above)
Comment (charlie-tf) [reply src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx:55]: oh! okay, i felt it was more organized to put everything I could up into `makeStyles`, but if there is a technical reason to prefer `useTheme` and `sx` then I'm happy to change this back and stick with that convention.

File: src/theme.ts (hunk 3 above)
Comment (charlie-tf) [reply src/theme.ts:6]: I actually wasn't sure about this, as I could not see either `TerrawareThemeOptions` or `TerrawareThemeOptions2` exported from those files, but I figured if it compiled successfully then it must be happening where I could not see.

Comment (karthikbtf) [reply src/theme.ts:6]: Oh it's just importing the default-export from those files, the naming here was intentional (my comments hopefully adds some context).. the main intention of this import is to get the theme options defined in memory - this is purely for typescript to stop complaining. if there's a better way to do this, we should.. maybe something for later.

Comment (charlie-tf) [reply src/theme.ts:6]: Ohh, gotcha 🎉

**Review by karthikbtf — APPROVED** [review]
🎉 great!!

**Pull request merged**